bingosync-app/templates/bingosync/bingosync.html

```diff
@@ -88,7 +93,30 @@ <h4 style="display: inline-block; position: absolute; right: 0; bottom: 0;">
 
         // hook up chat contents toggling
         initializeChatSettings($("#chat-settings"), $("#bingo-chat"));
+
+        // collapse event for the room settings panel
+        // I put it here becuase the template is loaded each time a new card is created
+        $("#room-settings-collapse").on("mousedown", function() {
```

Comment:
Minimizing room settings doesn't seem to work any more after I generate a new card. Maybe this `.on("mousedown", ...)` should go in refreshSettingsPanel?

Reply:
good catch